### PR TITLE
Feat standard load profile

### PR DESCRIPTION
**Pull request by missgreenwood**
Parse a standard load profile in csv format and write it to mybuzzn.db.

### CODE REVIEW

File: util/load_profile.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import csv
```
Comment (clemensmanert) [inline util/load_profile.py]: seems to be duplicate of util/parse_standard_load_profile.py

File: util/parse_standard_load_profile.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    database mybuzzn.db. """
+
+    try:
+        csvfile_path = parse_args().get('filepath')
+        with open(csvfile_path) as csvfile:
+            tbl_reader = csv.reader(csvfile, delimiter=',')
+            next(tbl_reader, None)
+            session = create_session()
+            for row in tbl_reader:
+                date = row[0]
```
Comment (clemensmanert) [inline util/parse_standard_load_profile.py]: please check if it is a valid date

File: util/parse_standard_load_profile.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                time = row[1]
```
Comment (clemensmanert) [inline util/parse_standard_load_profile.py]: and a valid time

File: models/loadprofile.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+from util.database import db
+
+
+class LoadProfileEntry(db.Model):
+    """ Represents one entry of a standard load profile. """
+
+    __tablename__ = 'loadprofile'
+    date = db.Column(db.String(33), primary_key=True)
```
Comment (clemensmanert) [inline models/loadprofile.py:8]: can be a date type

File: models/loadprofile.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    time = db.Column(db.String(33), primary_key=True)
```
Comment (clemensmanert) [inline models/loadprofile.py:9]: and a time type

**Review by clemensmanert — COMMENTED** [review]
util/parse_standard_load_profile.py and util/load_profile.py seem to be duplicates

File: util/load_profile.py (hunk 1 above)
Comment (missgreenwood) [reply util/load_profile.py]: Right, I will delete the other file!

**Pull request merged**